modules/psap-special-resource-operator-using-CRD.adoc

```diff
@@ -0,0 +1,451 @@
+                                                securityContext:
+                                                    privileged: true
+----
+... For a disconnected environment:
++
+[source,yaml]
+----
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+    annotations:
+        policy.open-cluster-management.io/categories: CM Configuration Management
+        policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+        policy.open-cluster-management.io/standards: NIST SP 800-53
+    name: version-claim-policy
+    namespace: default
+spec:
+    remediationAction: enforce
+    disabled: false
+    policy-templates:
+        - objectDefinition:
+            apiVersion: policy.open-cluster-management.io/v1
+            kind: ConfigurationPolicy
+            metadata:
+                name: version-claim-policy-config
+            spec:
+                remediationAction: enforce
+                severity: low
+                namespaceselector:
+                    exclude:
+                        - kube-*
+                    include:
+                        - '*'
+                object-templates:
+                    - complianceType: musthave
+                        objectDefinition:
+                            apiVersion: cluster.open-cluster-management.io/v1alpha1
+                            kind: ClusterClaim
+                            metadata:
+                                name: osimage.openshift.io
+                            spec:
+                                value: '{{ (lookup "config.openshift.io/v1" "ClusterVersion" "" "version").status.desired.image }}' <1>
+----
+<1> This ACM Policy creates the necessary cluster claim and keeps it in sync with changes if the spoke’s OCP version is upgraded.
+.. Save this YAML template for the placement of policies in the `templates` directory as `0003-policy.yaml`.
+... For a connected environment:
++
+[source,yaml]
+----
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+    name: {{.Values.specialResourceModule.metadata.name}}-placement
+spec:
+    clusterConditions:
+    - status: "True"
+        type: ManagedClusterConditionAvailable
+    clusterSelector:
+        matchExpressions:
+            - key: name
+                operator: NotIn
+                values:
+                    - local-cluster
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+metadata:
+    name: {{.Values.specialResourceModule.metadata.name}}-binding
+placementRef:
+    apiGroup: apps.open-cluster-management.io
+    kind: PlacementRule
+    name: {{.Values.specialResourceModule.metadata.name}}-placement
+subjects:
+    - apiGroup: policy.open-cluster-management.io
+        kind: Policy
+        name: policy-{{.Values.specialResourceModule.metadata.name}}-ds
```

Comment:
Should be at the same level as `apiGroup`

Reply:
corrected